tests/components/yr/test_sensor.py

```diff
@@ -116,5 +117,5 @@
     assert state.state == "0.0"
 
     state = hass.states.get("sensor.yr_wind_speed")
-    assert state.attributes.get("unit_of_measurement") == "m/s"
+    assert state.attributes.get("unit_of_measurement") == f"m/{TIME_SECONDS}"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5054188)